build-css.cjs

```diff
@@ -0,0 +1,7 @@
+const sass = require('sass')
```

Comment:
We will find a proper solution for this (maybe swap tsc with tsup?), but I'm ok with this hack for the moment. Discussion here: https://github.com/seamapi/react/discussions/36

Reply:
Exactly. This was definitely just to get something out the door vs. messing with webpack configs for 3 days.